src/test/java/com/google/api/codegen/testdata/discoveries/go/go_adexchangebuyer.v1.4.json.baseline

```diff
@@ -34,25 +32,24 @@ func main() {
   }
 
   // The account id
-  id := int64(0)
-  // TODO: Change placeholder value above.
+  id := int64(0)  // TODO: Update placeholder value.
 
   resp, err := adexchangebuyerService.Accounts.Get(id).Context(ctx).Do()
   if err != nil {
     log.Fatal(err)
   }
 
   // TODO: Change code below to process the `resp` object:
-  fmt.Printf("%#v\n", resp)
+  log.Printf("%#v\n", resp)
```

Comment:
I must have overlooked these getting changed back to `log` --- I think it should stay as `fmt` since `log` writes to stderr by default, and that doesn't seem appropriate for response output.